test/test_variant.py

```diff
@@ -41,6 +41,9 @@ def test_insertion_shared_prefix():
     eq_(variant.start, 11)
     eq_(variant.end, 11)
     eq_(variant.short_description, "chr1 g.11_12insT")
+    assert variant.is_indel
```

Comment:
Hence being a nit; http://nose.readthedocs.org/en/latest/testing_tools.html#nose.tools.ok_ "Saves 3 whole characters!"

Completely up to you.